cmp/contextmenu/ContextMenuItem.js

```diff
@@ -20,14 +20,14 @@ export class ContextMenuItem {
 
     /**
      * @param {string} text - label to be displayed.
-     * @param {Object} icon - optional icon to be displayed.
-     * @param {function} action - Executed when the user clicks the menuitem.
-     * @param {Object[]} items - child menu items.
-     * @param {boolean} disabled - true to disable this item.
-     * @param {boolean} hidden - true to hide this item.
-     * @param {function} prepareFn - function of the form (item, record, selection) => {}
-     *          The prepareFn is a callback that is triggered before each time the menuitem is shown.
-     *          It can be used to modify the menuitem based on the record / selection.
+     * @param {Object} [icon] - optional icon to be displayed.
+     * @param {function} [action] - Executed when the user clicks the menuitem.
+     * @param {Object[]} [items] - child menu items.
+     * @param {boolean} [disabled] - true to disable this item.
+     * @param {boolean} [hidden] - true to hide this item.
+     * @param {function} [prepareFn] - function of the form (item, record, selection) => {}
+     *      The prepareFn is a callback that is triggered before each time the menuitem is shown.
+     *      It can be used to modify the menuitem based on the record / selection.
```

Comment:
There is a specific way to document the signature for callbacks, but not sure if we're drinking that Kool Aid. http://usejsdoc.org/tags-param.html#callback-functions  @lbwexler ?

Reply:
Would like to skip.  A function is always in some sense a "callback", and we have the function type and fn suffix already.